recipes/pyqode.qt/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pyqode.qt" %}
+{% set version = "2.10.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cfe97d85b210f0d77663c9df47bcb25bbb7cbbe2d9f58c0a9fd94e5e1d7daa2f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - pyqt
+  run:
+    - pyqt
```

Comment:
You might need `setuptools` as a `run` dependency since `pkg_resources` is imported:

https://github.com/pyQode/pyqode.qt/blob/master/pyqode/__init__.py#L7